### DISCUSSION TITLE
[Esame 17 luglio 2024] Esercizio 2

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#581] [Esami M-Z]
![photo_5906649595277066443_y](https://github.com/user-attachments/assets/1c314f04-8184-40e5-ab77-5a8fd8e0658b)

<details>
  <summary>soluzione del prof</summary>
  <br>
  <img src="https://github.com/user-attachments/assets/df45f842-2608-43c1-a196-22904216c030" alt="soluzione"  />
  <img src="https://github.com/user-attachments/assets/e71a66e5-5286-450f-a319-1ad00873b675" alt="soluzione"  />

</details>

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/2f638a39-5fff-42dc-9ec6-2a5ea3a2d54f)

**Comment by niccolo-fato** [comment]
<img width="1280" height="1105" alt="image" src="https://github.com/user-attachments/assets/02c0f70e-0a2a-41e8-a314-0a75c2cf2108" />
